client/src/components/application/Examine/CompName.vue

```diff
@@ -181,6 +181,15 @@
       userId() {
         return this.$store.getters.userId;
       },
+      userIsAnExaminer() {
+        return this.$store.getters.userHasApproverRole;
+      },
+      userCanEdit() {
+        return this.$store.getters.userHasEditRole;
+      },
+      canCancel() {
+        return this.userIsAnExaminer || (this.userCanEdit && ['DRAFT'].indexOf(this.currentState) > -1);
```

Comment:
Restricting this to DRAFT - is that a requirement of this task? I don't think it is. I believe an NR can be cancelled even after decision is made as long as it's not expired or consumed. Yes, this doesn't logically make sense but is the current functionality and doesn't have to do with roles, so should not be changed in this task.
I *did* see something mentioned in the original task re. admins perhaps not having full cancel ability. If that's been defined with Kaine let me know.

Reply:
Man, there have been so many comments in this ticket that it's really hard to keep everything straight.

Reply:
Okay when in doubt (which I am) let's not change the functionality. That's outside the scope of the ticket re. roles (although I realize that's a fairly iffy statement with this task :P). But let's err on the side of caution.